src/saltext/vmware/modules/vmc_org_users.py

```diff
@@ -0,0 +1,441 @@
+        (Optional) Specifies the index that the set of results will begin with
+
+    service_definition_id: String
+        (Optional) Service definition id used to filter users having access to the service.
+
+    verify_ssl
+        (Optional) Option to enable/disable SSL verification. Enabled by default.
+        If set to False, the certificate validation is skipped.
+
+    cert
+        (Optional) Path to the SSL client certificate file to connect to VMC Cloud Console.
+        The certificate can be retrieved from browser.
+
+    """
+
+    log.info("Retrieving Org users for the organization %s", org_id)
+    api_base_url = vmc_request.set_base_url(hostname)
+    api_url = "{base_url}csp/gateway/am/api/v2/orgs/{org_id}/users".format(
+        base_url=api_base_url, org_id=org_id
+    )
+
+    params = vmc_request._filter_kwargs(
+        allowed_kwargs=[
+            "expandProfile",
+            "includeGroupIdsInRoles",
+            "serviceDefinitionId",
+            "pageStart",
+            "pageLimit",
+        ],
+        expandProfile=expand_profile,
+        includeGroupIdsInRoles=include_group_ids_in_roles,
+        serviceDefinitionId=service_definition_id,
+        pageStart=page_start,
+        pageLimit=page_limit,
+    )
+    return vmc_request.call_api(
+        method=vmc_constants.GET_REQUEST_METHOD,
+        url=api_url,
+        refresh_key=refresh_key,
+        authorization_host=hostname,
+        description="vmc_org_users.list",
+        params=params,
+        verify_ssl=verify_ssl,
+        cert=cert,
+    )
+
+
+def search(
+    hostname, refresh_key, org_id, user_search_term, expand_profile=None, verify_ssl=True, cert=None
+):
+    """
+    Search users in organization having username, firstName, lastName or email which "contains" search term.
+    e.g. search for "test" will return test@vmware.com if test@vmware.com is part of the organization.
+    Search results limited to first 20 results. Please refine the search term for accurate results.
+    Organization members will receive basic user information.
+    Organization owners and Service Owners (for organizations that have access to the service) will additionally receive role details of the users.
+
+    Please refer the `Search Org Users documentation <https://developer.vmware.com/apis/csp/csp-iam/latest/csp/gateway/am/api/orgs/orgId/users/search/get/>`_ to get insight of functionality and input parameters
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt <minion id> vmc_org_users.search hostname=console.cloud.vmware.com org_id=org_id  ...
+
+    hostname
+        The host name of CSP
+
+    refresh_key
+        API Token of the user which is used to get the Access Token required for VMC operations
+
+    org_id
+        The Id of organization for which user list is retrieved
+
+    user_search_term: String
+        (Required) The string to be searched within email or firstName or lastName or username
+
+    expand_profile: String
```

Comment:
**suggestion** expand_profile should also be bool here

Reply:
updated properly